.github/workflows/ci.yaml

```diff
@@ -14,9 +14,10 @@ jobs:
     runs-on: ubuntu-latest
     services:
       goaws:
-        image: motoserver/moto
+        image: motoserver/moto:latest
         ports:
           - 5000:5000
+        options: --env MOTO_PORT=5000
```

Comment:
Is it to fix the e2e test? if so, please add some comments on how it fixes and fixes what issue. Same comment for [config/apps/moto/moto.yaml](https://github.com/numaproj-contrib/aws-sqs-source-go/pull/13/files#diff-907f5d33f76166f1e1fe96ab2588043565c6fd90eadb7534142b0300ad6019c3)